ui/src/data-services/hooks/identifications/useCreateIdentifications.ts

```diff
@@ -0,0 +1,20 @@
+import { IdentificationFieldValues } from './types'
+import { useCreateIdentification } from './useCreateIdentification'
+
+export const useCreateIdentifications = (
+  params: IdentificationFieldValues[]
+) => {
+  const { createIdentification, isLoading, isSuccess } =
+    useCreateIdentification()
+
+  return {
+    isLoading,
+    isSuccess,
+    createIdentifications: async () => {
```

Comment:
This is great! The async really pays off here. I see all 16 requests fire off at once and the backend seems to respond quickly.